x_axis_controller:
  type: effort_controllers/JointEffortController
  joint: x_axis_joint
y_axis_controller:
  type: effort_controllers/JointEffortController
  joint: y_axis_joint
  pid: {p: 500.0, i: 0.0, d: 0.1, i_clamp: 1.0}
yaw_controller:
  type: effort_controllers/JointVelocityController
  joint: joint1
  pid: {p: 500.0, i: 0.0, d: 0.1, i_clamp: 0.0}
up_down_controller:
  type: effort_controllers/JointEffortController
  joint: joint2
  pid: {p: 100000.0, i: 0.0, d: 0.1, i_clamp: 1.0}
pitch_controller:
  type: effort_controllers/JointVelocityController
  joint: joint3
  pid: {p: 500.0, i: 0.0, d: 0.1, i_clamp: 0.0}
roll_controller:
  type: effort_controllers/JointVelocityController
  joint: joint4
  pid: {p: 500.0, i: 0.0, d: 0.1, i_clamp: 0.0}
gravity_compensation:
  type: "robot_controllers/GravityCompensation"
  root: "base_link"
  tip: "plat_base"
  autostart: true